### PR TITLE
feat: option to add space between behind & ahead upstream

### PR DESCRIPTION
**Pull request by marcoSven**
## Purpose
Adding a space between behind & ahead upstream is currently not possible

## Approach
Adding an option to add a single space

Before 
Screenshot 2024-04-15 at 15 46 38

After
Screenshot 2024-04-15 at 15 47 17

### CODE REVIEW

**Comment by arl** [comment]
Can't you obtain the same outcome by adding a space in front of the behind symbols in your gitmux file?

**Comment by marcoSven** [comment]
We only want to add a space between ahead and behind count. 

Adding a space in front would add an undesired extra space.

"Ahead""\sBehind"

"\sBehind" <- undesired space

File: README.md [hunk 1]
```diff
@@ -264,13 +265,14 @@ layout: [branch, "|", flags, "|", stats]
...
-| `hide_clean`     | Hides the clean flag entirely                              |      `false`       |
-| `swap_divergence`| Swaps order of behind & ahead upstream counts              |      `false`       |
+| Option                    | Description                                                |      Default       |
+| :------------------------ | :--------------------------------------------------------- | :----------------: |
+| `branch_max_len`          | Maximum displayed length for local and remote branch names |   `0` (no limit)   |
+| `branch_trim`             | Trim left or right end of the branch (`right` or `left`)   | `right` (trailing) |
+| `ellipsis`                | Character to show branch name has been truncated           |        `…`         |
+| `hide_clean`              | Hides the clean flag entirely                              |      `false`       |
+| `swap_divergence`         | Swaps order of behind & ahead upstream counts              |      `false`       |
+| `space_between_divergence`| Add space between behind & ahead upstream counts           |      `false`       |
```
[the remark this answers is not in the document]
Comment (marcoSven) [reply README.md]: Doing the changes on my phone - hope this is right now

File: .gitmux.yml [hunk 2]
```diff
@@ -83,3 +83,4 @@ tmux:
         hide_clean: false
         # Swaps order of behind & ahead upstream counts - "↓·1↑·1" -> "↑·1↓·1"
         swap_divergence: false
+        divergence_space: false
```
Comment (arl) [inline .gitmux.yml:86]: This misses a doc string. You can use the same one as the readme.
The content of this file is output for `gitmux -printfg`

**Review by arl — APPROVED** [review]
Sorry for the long delay between review.
There's just a last thing and we're good 
Thank you.

**Comment by arl** [comment]
@marcoSven I'm closing this in favor of #117 which got your code and some additional commits I've made on top of them.

thank you!

**Pull request closed without merging**